src/components/TopAppBar/TopAppBar.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { FC } from 'react';
+import Styles from './TopAppBar.style';
+import { DEFAULT_BG_COLOR, DEFAULT_SEARCH_COLOR, TopAppBarProps } from './TopAppBar.types';
+import { SidebarMenuIcon } from './components';
+import LogoWrapper from './components/Logo.wrapper';
+import TextField from '../TextField';
+import Icon from '../Icon';
+import { pickTextColorFromSwatches } from '../../theme/palette';
+import Menu from '../Menu';
+
+const TopAppBar: FC<TopAppBarProps> = ({
+  bgColor = DEFAULT_BG_COLOR,
+  searchBgColor = DEFAULT_SEARCH_COLOR,
+  searchLabel = 'Search',
+  logoIcon,
+  onMenuIconClick,
+  additionalTools,
+  userMenu,
+  onSearchHandler,
+}) => {
+  const { type, variant } = searchBgColor;
+  const { items, userAvatar, userName, onSelect } = userMenu;
+  const searchProps = { onChange: onSearchHandler };
+
+  return (
+    <div
+      role="banner"
+      aria-label="Top Application Banner"
+      css={Styles.topAppBarWrapper({ bgColor })}
+    >
+      <div css={Styles.mainSection}>
+        <SidebarMenuIcon onMenuIconClick={onMenuIconClick} />
+        <LogoWrapper logoIcon={logoIcon} />
+        <div css={Styles.searchWrapper}>
+          <TextField
+            label={searchLabel}
```

Comment:
label here must change to placeholder e.g `placeholder={searchLabel}`. The label does the animation material like which we dont want at the top bar